frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -129,6 +132,14 @@ private boolean isManagementReq(String[] segments) {
                 || endpointMap.containsKey(segments[1]);
     }
 
+    private boolean isModeExplicit() {
+        if (Modelmode == null) {
+            ConfigManager configManager = ConfigManager.getInstance();
+            Modelmode = configManager.getModelControlMode();
+        }
+        return Modelmode;
+    }
+
```

Comment:
This can be moved to the config manager `getModelControlMode`, so we can just query the model control mode here from the config manager and not have to store the configuration value.